[33mWARNING [0m urllib3.connectionpool:connectionpool.py:751 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x041064A8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/ad17877d6b96d2d026d70227707d2b3c/elements
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:751 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x04106550>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/ad17877d6b96d2d026d70227707d2b3c/elements
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:751 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x041067D8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/ad17877d6b96d2d026d70227707d2b3c/elements